main.py

```diff
@@ -163,8 +163,12 @@ def actuators(self, message, nick, pm):
             elif "!meow" in message_lower:
                 result = random_cat.random_cat_pic()
                 self.send_msg(result, nick, pm)
-            
-            elif "!jesus" in message_lower:
+
+            elif "!catfact" in message_lower:
+                result = random_cat_fact.random_cat_facts()
+                self.send_msg(result, nick, pm)
+ 
+             elif "!jesus" in message_lower:
```

Comment:
IndentationError: unindent does not match any outer indentation level
```suggestion
           elif "!jesus" in message_lower:
```

NameError: name 'random_cat_fact' is not defined
from modules import random_cat_fact